analyzers/its/expected/Ember-MM/generic.EmberCore.NMT-{84B2143A-D04F-4262-923D-21AEDF86E2B7}-S2259.json

```diff
@@ -12,6 +155,123 @@
 "endColumn":  34
 }
 }
+},
+{
+"id":  "S2259",
+"message":  "'c.values.FirstOrDefault(Function(y) y.label = saved)' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  960,
+"startColumn":  42,
+"endLine":  960,
+"endColumn":  94
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'p' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  998,
+"startColumn":  13,
+"endLine":  998,
+"endColumn":  14
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'p.values.FirstOrDefault(Function(y) y.label = s)' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  998,
+"startColumn":  23,
+"endLine":  998,
+"endColumn":  71
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'conf.Files.FirstOrDefault(Function(y) y.Process = True AndAlso y.Type = "movieindex")' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  1498,
+"startColumn":  71,
+"endLine":  1498,
+"endColumn":  156
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'conf.Files.FirstOrDefault(Function(y) y.Process = True AndAlso y.Type = "movieindex")' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  1498,
+"startColumn":  168,
+"endLine":  1498,
+"endColumn":  253
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'conf.Files.FirstOrDefault(Function(y) y.Process = True AndAlso y.Type = "tvindex")' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  1561,
+"startColumn":  71,
+"endLine":  1561,
+"endColumn":  153
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'conf.Files.FirstOrDefault(Function(y) y.Process = True AndAlso y.Type = "tvindex")' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  1561,
+"startColumn":  165,
+"endLine":  1561,
+"endColumn":  247
+}
+}
+},
+{
+"id":  "S2259",
+"message":  "'conf.Params.FirstOrDefault(Function(y) y.name = dgvSettings.Rows(e.RowIndex).Cells(0).Value.ToString)' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.NMT\dlgNMTMovies.vb",
+"region":  {
+"startLine":  1695,
+"startColumn":  29,
+"endLine":  1695,
+"endColumn":  130
+}
+}
+},
```

Comment:
TP `description` is a field.
https://github.com/SonarSource/sonar-dotnet/blob/145047ee1ed01316ef730427c7af96f0602ee2d8/analyzers/its/sources/Ember-MM/Addons/generic.EmberCore.NMT/dlgNMTMovies.vb#L1695
https://github.com/SonarSource/sonar-dotnet/blob/1711a5f773013925e079d11e32bf0a3f592a6706/analyzers/its/sources/Ember-MM/Addons/generic.EmberCore.NMT/Module.NMT.vb#L213-L218